modules/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpConnectionManager.java

```diff
@@ -270,7 +272,9 @@ private void validateConnectorStartup(ConnectorStartupSynchronizer startupSyncer
     }
 
     private boolean isHTTPTraceLoggerEnabled() {
-        return System.getProperty(BLogManager.HTTP_TRACE_LOGGER) != null ? true : false;
+        // TODO: Take a closer look at this since looking up from the Config Registry here caused test failures
+        return ((BLogManager) LogManager.getLogManager()).getPackageLogLevel(
+                org.ballerinalang.logging.util.Constants.HTTP_TRACE_LOG) == BLogLevel.DEBUG;
```

Comment:
is this DEBUG or TRACE ?

Reply:
It's DEBUG. The name trace logs doesn't have anything to do with the log level

Reply:
We have Trace logger. I think this should goto trace.